README.md

```diff
@@ -283,6 +283,14 @@ TCP может отсылать данные вне основного пото
 * Linux: `make`
 * Windows: `make windows CC=x86_64-w64-mingw32-gcc`
 
+------
+### Docker
+
+```sh
+docker build ./ -t byedpi:latest
+docker run -p 1080:1080 -ti byedpi:latest ciadpi --disorder 1 --split 1 
```

Comment:
`split` is meaningless in the same position as `disorder` (it won't send anything).